server/services/Mailer.js

```diff
@@ -3,24 +3,22 @@ const helper = sendgrid.mail;
 const keys = require('../config/keys');
 
 class Mailer extends helper.Mail {
-  constructor({ subject, recipients }, content) {
+  constructor(survey, content) {
     super();
```

Comment:
This was the confusing argument destructor I was talking about. This constructor, as we saw just above, takes a survey as it's first argument. However, you wouldnt' know that just by looking at it due to the argument deconstructions.

I think it's much easier to ready by simply saying a survey object is passed in.